docs/howto/get-started-with-charm-testing.md

```diff
@@ -91,60 +91,48 @@ A 'live', deployed Juju application will have access to all the inputs we discus
  
 You will notice that the starting point is typically always an event. A charm doesn't do anything unless it's being run, and it is only run when an event occurs. So there is *always* an event context to be mocked. This has important consequences for the unit-testing framework, as we will see below.
 
-### The harness
+### The framework
 
-In the charming world, unit testing means using Harness.
+In the charming world, unit testing means state-transition testing.
 
-> See more [`ops.testing.Harness`](https://ops.readthedocs.io/en/latest/harness.html#ops.testing.Harness)
+> See more [`ops.testing`](https://ops.readthedocs.io/en/latest/reference/ops-testing.html)
 
-Harness is the 'mocker' for most inputs and outputs you will need. Where a live charm would gather its input through context variables and calls to the Juju api (by running the hook tools), a charm under unit test will gather data via a mocked backend managed by Harness. Where a live charm would produce output by writing files to a filesystem, Harness exposes a mock filesystem the charm will be able to interact with without knowing the difference. More specific outputs, however, will need to be mocked individually.
+`State` is the 'mocker' for most inputs and outputs you will need. Where a live charm would gather its input through context variables and calls to the Juju api (by running the hook tools), a charm under unit test will gather data via a mocked backend managed by the testing framework. Where a live charm would produce output by writing files to a filesystem, `Context` and `Container` expose a mock filesystem the charm will be able to interact with without knowing the difference. More specific outputs, however, will need to be mocked individually.
 
-A typical test with Harness will look like this:
+A typical test will look like this:
  
 - set things up:
   - set up the charm and its metadata
-  - set up the harness
-  - mock any 'output' callable that you know would misfire or break (e.g. a system call -- you don't want a unittest to reboot your laptop)
-  - configure the charm
-  - mock any relation data
+  - set up the context
+  - mock any 'output' callable that you know would misfire or break (e.g. a system call -- you don't want a unit test to reboot your laptop)
+  - set up the Juju state in which the event will fire, including config and relation data
  - **mock an event**
```

Comment:
I wonder if we can improve this "mock an event" language to use "Scenario"-specific terms, like "trigger an event using `Context.run`". We're not really "mocking" an event.